pkg/globalcontext/event/reason.go

```diff
@@ -1,8 +1,5 @@
 package event
 
 const (
-	ReasonResourceListFailure = "FailedToList"
-	ReasonAPICallFailure      = "FailedToCallAPI"
```

Comment:
we don't need them anymore ?

Reply:
Nope. I used to use them for events. But I found out the Kyverno uses only four Reasons.

Reply:
Yeah, they were used to build the events reason for the contextEntry but those aren't aligned with the event reasons of the other policies.